source/usage-examples/insertOne.txt

```diff
@@ -25,11 +25,8 @@ can also pass a callback method as an optional third parameter.
 For more information on this method, see the
 :node-api-4.0:`insertOne() API documentation </classes/collection.html#insertone>`.
 
-The ``insertedId`` field of this object is the ``_id`` of the
-inserted document. If the operation fails to create a document, the
-``insertedCount`` field of this object has a value of ``0``. If the
-operation creates a document, the ``insertedCount`` field of this object
-has a value of ``1``.
+If the operation creates a document, the ``insertedId`` field of this
+object is the ``_id`` of the inserted document. 
```

Comment:
I think there's a grammatical issue here. The operation appends _id value of the inserted document to the object as the insertedId field, right?
Currently, the effect of this sentence doesn't seem connected to the cause. Here's something similar in structure to this sentence:
"If Chris goes to the grocery store, the bag is full of chips and salsa."
Instead, you might say:
"If Chris goes to the grocery store, he purchases a bag full of chips and salsa."